gapis/api/vulkan/resources.go

```diff
@@ -1471,34 +1471,15 @@ func (p GraphicsPipelineObjectʳ) colorBlending() *api.Stage {
 		Rows:    targetRows,
 	}
 
-	renderAttachments := p.RenderPass().AttachmentDescriptions()
-	attachmentRows := make([]*api.Row, renderAttachments.Len())
-	for i, index := range renderAttachments.Keys() {
-		attach := renderAttachments.Get(index)
-
-		attachmentRows[i] = &api.Row{
-			RowValues: []*api.DataValue{
-				api.CreateEnumDataValue("VkFormat", attach.Fmt()),
-				api.CreateBitfieldDataValue("VkSampleCountFlagBits", attach.Samples(), VkSampleCountFlagBitsConstants(), API{}),
-				api.CreateEnumDataValue("VkAttachmentLoadOp", attach.LoadOp()),
-				api.CreateEnumDataValue("VkAttachmentStoreOp", attach.StoreOp()),
-				api.CreateEnumDataValue("VkAttachmentLoadOp", attach.StencilLoadOp()),
-				api.CreateEnumDataValue("VkAttachmentStoreOp", attach.StencilStoreOp()),
-				api.CreateEnumDataValue("VkImageLayout", attach.InitialLayout()),
-				api.CreateEnumDataValue("VkImageLayout", attach.FinalLayout()),
-			},
-		}
-	}
-
-	attachTable := &api.Table{
-		Headers: []string{"Format", "Samples", "Load Op", "Store Op", "Stencil Load Op", "Stencil Store Op", "Init Layout", "Final Layout"},
-		Rows:    attachmentRows,
-	}
+	renderPassList := &api.KeyValuePairList{}
+	renderPassHandle := p.RenderPass().VulkanHandle()
+	renderPassPath := path.NewField("RenderPasses", resolve.APIStateAfter(path.FindCommand(cmd), ID)).MapIndex(renderPassHandle)
```

Comment:
Leave a TODO here to use Link()?

Reply:
We decided that Link() is redundant since I already have the api.GlobalState

Reply:
OK.